detect/detect.go

```diff
@@ -194,6 +194,9 @@ func (d *Detector) Detect(fragment Fragment) []report.Finding {
 	}
 
 	for _, rule := range d.Config.Rules {
+		if !rule.Enabled {
+			continue
+		}
```

Comment:
It seemed easiest to simply skip over the rule, rather than trying to remove it from `Rules`, `OrderedRules`, and `Keywords` in Config.